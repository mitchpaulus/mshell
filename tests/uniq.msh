[ 3 1 2 1 3 2 ] uniq str wl
[ "b" "a" "b" "c" ] uniq str wl
[2025-05-01 2025-05-02 2025-07-07 2025-05-02] uniq str wl
["a" a] uniq str wl # Literals treated as strings
[a "a"] uniq str wl # Literals treated as strings, should be converted to string
[1.4 1.2 1.4 1.3] uniq str wl
[1 "a" 1 "a" 2025-07-07 2025-07-07 0.1 0.1] uniq str wl # Mix and match
["a" "a" `a` `a`] uniq str wl # Paths and strings treated separate
