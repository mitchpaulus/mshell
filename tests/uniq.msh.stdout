[3 1 2]
["b" "a" "c"]
[2025-05-01 2025-05-02 2025-07-07]
["a"]
["a"]
[1.4 1.2 1.3]
[1 "a" 2025-07-07 0.1]
["a" `a`]
